### PR TITLE
Updated uncover link to use block tab over extrinsic tab

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Fixes issue reported by @Amy-Centrality 

There's a report of transaction completed on CENNZX, but then UNcover shows it as fail. Any idea why this would be? https://www.uncoverexplorer.com/extrinsic/0xcab35dfbabbce9c8d6710e3c555fc366c75f11b0f6ec04335c23e8ae7299e252

### CODE REVIEW

File: src/components/Dialog/TxDialog/TxDialogBody.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import SummaryFee from '../../AdvancedSetting/SummaryFee';
 
 type AssetSwapParams = [number, number, Amount, Amount];
 
-const getCennzScanURL = txHash => `https://www.uncoverexplorer.com/extrinsic/${txHash}`;
+const getCennzScanURL = txHash => `https://www.uncoverexplorer.com/block/${txHash}`;
```
Comment (holygits) [inline src/components/Dialog/TxDialog/TxDialogBody.tsx:15]:
```suggestion
const getCennzScanURL = blockHash => `https://www.uncoverexplorer.com/block/${blockHash}`;
```

File: src/components/Dialog/TxDialog/TxDialogBody.tsx [hunk 2]
```diff
@@ -95,7 +95,7 @@ const BodyForFinalised: FC<BodyForFinalisedProps> = ({
                           Amount.ROUND_UP
                       )}  ${assetInfo[feeExchangeResult.assetId].symbol}`
                     : ''}
-                <br /> Transaction hash:
+                <br /> Block hash:
                 <ExternalLink url={getCennzScanURL(txHash)} text={txHash} />
```
Comment (holygits) [inline src/components/Dialog/TxDialog/TxDialogBody.tsx:99]:
```suggestion
                <ExternalLink url={getCennzScanURL(blockHash)} text={blockHash} />
```

**Review by holygits — COMMENTED** [review]
not sure I understand this, have we always been returning block hash incorrectly instead of tx hash?

**Comment by KarishmaBothara** [comment]
> not sure I understand this, have we always been returning block hash incorrectly instead of tx hash?

Unfortunately yes... did not notice this as uncover was running for Azalea/Nikau...
Have updated the code to return extrinsic hash... Verified by checking hash value on cennznet ui for local node...
Screen Shot 2021-07-05 at 2 16 25 PM

File: src/components/Dialog/TxDialog/TxDialogBody.tsx (hunk 1 above)
Comment (KarishmaBothara) [reply src/components/Dialog/TxDialog/TxDialogBody.tsx]: updated the code to return txHash

File: src/redux/epics/txDialog/submitTransaction.epic.ts [hunk 3]
```diff
@@ -161,7 +158,7 @@ export const submitLiquidityEpic = (
                                 switchMap(({events, status}) => {
                                     if (status.isInBlock) {
                                         return of(
-                                            updateTxHash(status.asInBlock.toString()),
+                                            updateTxHash(tx.hash.toString()),
```
Comment (holygits) [inline src/redux/epics/txDialog/submitTransaction.epic.ts:161]: 🤦

**Review by holygits — APPROVED** [review]

**Pull request merged**